src/solvers/optimize.jl

```diff
@@ -80,7 +80,7 @@ function optimize(optprob::OptimizationProblem,method=LineSearch(Newton()),optio
 end
 #build scalar objective -> Optimization Problem
 function optimize(scalarobj::ScalarObjective,x0,method=LineSearch(Newton()),options=OptimizationOptions();bounds = nothing)
-    optprob = OptimizationProblem(scalarobj,inplace = (x0 isa number),bounds = bounds)
+    optprob = OptimizationProblem(scalarobj,inplace = (x0 isa Number)) #,bounds = bounds)
```

Comment:
Hey Andres, 
I managed to make the tpd minimization to work but I had to change number -> Number here and also left out the bounds for the minimization. It kept telling me that OptimizationProblem only accepts the objective function and the inplace boolean.

Reply:
i think that is the same error as the NeqProblem before, let me check

Reply:
also, got isofugacity working with the saturation pressure interface:
```
julia> using Clapeyron: IsoFugacitySaturation

julia> @inferred saturation_pressure(pr,373.15,IsoFugacitySaturation(p0=1e5))
(96099.38979351871, 2.2674781912892933e-5, 0.03201681565699547)

julia> @inferred saturation_pressure(pr,373.15,IsoFugacitySaturation()) #calculates p0 via psat_init
(96099.38979351871, 2.2674781912892933e-5, 0.03201681565699359)

julia> saturation_pressure(pr,373.15,ChemPotVSaturation()) #Clapeyron default
(96099.38979351855, 2.2674781912892906e-5, 0.03201681565699426)
```